RSSParser/RSSParser.m

```diff
@@ -199,6 +211,27 @@ -(NSURL *)getNSURLFromString:(NSString *)string {
     }
 }
 
+-(NSDate *)getDateFromString:(NSString *)string {
+    NSDate *date = nil;
+    if (string) {
+        date = [_formatter dateFromString:string];
+        if (!date) {
+            date = [_ISO8601formatter dateFromString:string];
+        }
+        if (!date) {
+            NSRange range = [string rangeOfString:@" " options:NSBackwardsSearch];
+            NSString *threeLetterZone = [string substringFromIndex:range.location+1];
+            NSTimeZone *timeZone = [NSTimeZone timeZoneWithAbbreviation:threeLetterZone];
+            if (timeZone)
+            {
+                NSString *gmtTime = [string stringByReplacingOccurrencesOfString:threeLetterZone withString:@"GMT"];
```

Comment:
I don't know why, but for the life of me, I could not get an NSDate object if the `dateFormatString` contained "EST".

Even so, the example I found online used this method to convert a time to UTC and it seemed to work so...(shrug)